alpha-build-machinery/make/targets/openshift/crd-schema-gen.mk

```diff
@@ -0,0 +1,24 @@
+CRD_SCHEMA_GEN_TEMP ?=$(shell mktemp -d)
+
+# $1 - schema gen version
+# $2 - manifests directory
+# $3 - package APIs
+define codegen-crds
```

Comment:
What value does the macro have if it defines fixed targets? You can run it only once so the macro arguments could be fixed variables and you'd not need a macro to begin with.

Is there a usecase where you'd want to have 2 or more separate runs over a single run with multiple args?